pages/docs/integrations/grouparoo-calculated-property.mdx

```diff
@@ -1,24 +1,94 @@
 
-This guide will show you how to work with the Calculated Property Plugin to [create a custom Calculated Property](#create-calculated-property-property)
+## Overview
 
-<PluginDocsInstallation plugin="Calculated Property" />
+Grouparoo's Calculated Property Integration enables you to write a custom function to generate a new Property based upon other existing Grouparoo Record Properties. You may want to combine, aggregate, or manipulate properties from other Sources to create a new Property that suits your operational needs.
 
-<PluginDocsCreateApp plugin="Calculated Property" />
+### Sources
 
-<PluginDocsCalculatedPropertySource plugin="Calculated Property" />
+<table className="docTable">
+  <tablebody>
+    <tr>
+      <td>Source Name</td>
+      <td>Description</td>
+    </tr>
+    <tr>
+      <td>Calculated Property Import</td>
+      <td>Import and calculate Calculated Properties from other Sources.</td>
+    </tr>
+  </tablebody>
+</table>
 
-<ValidateAndApplyConfig headingLevel="h3" />
+## Setup
 
-## Calculated Property Next Steps
+To work with the Calculated Property Plugin, you must first install it in an existing Grouparoo project. In addition, you must have at least one other Source configured to make use of Calculated Properties.
 
-Once you have the Plugin installed, App and Source created, and Property configured, you are ready to [validate](/docs/cli/config#validate), [apply](/docs/cli/config#apply), then [import or export your data](/docs/cli/run#run)!
+## App
+
+To create Calculated Properties in Grouparoo, you [create an App](https://www.grouparoo.com/docs/config/apps).
+
+### Options
+
+Some types of Apps require information about how to connect to a service. This integration works a little differently than most and no additional options are required in order to proceed in configuring a Calculated Property as a Source.
+
+## Calculated Property Import
+
+A Calculated Property must be assigned a key, which you can think of as the Property’s name. You must also choose the data type for this property from a dropdown menu and indicate whether this Property is Unique or an Array.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/calculated-property/app/options.png"
+  alt="Calculated Property App Options"
+  width={733}
+  height={211}
+/>
+<br></br>
+<br></br>
+
+### Options
+
+The Calculated Property Source is a specific type of Source. It performs a calculation on properties derived from another source and gives you the ability to generate custom Properties using a JavaScript function in the Property configuration.
+
+Calculations resulting in a string property can be done using a JavaScript pattern similar to the following example for anything that would result in a string Property:
+
+```js
+()=>”{{{lastName}}}{{{firstName}}}”.toLowerCase()
+```
```

Comment:
In code examples, it's probably best to use "boring" quotes, eg: `"`

Reply:
changed it, thanks.